io_directory_example/settings/lens_system3_config.yml

```diff
@@ -26,7 +26,10 @@ model:
 
 lens_option:
   centroid_init: [0.04, -0.04]
-
+  gaussian_prior:
+    0 :
```

Comment:
I remember that we should aim for user friendliness and that the user should only have specific the name of the profile instead of its index. But I notice that for in the settings yml file, you can find

`lens_light_option: fix: { 0: { 'n_sersic': 4. } }`, 

so I was thinking to be consistent I would follow the same convention, any thoughts?

Reply:
For now, using index instead of name is fine. But, we need to make proper documentations for these conventions, so that it is clear to a new user how to set up the config files properly.